src/views/Player.vue

```diff
@@ -64,11 +63,10 @@ export default {
         'failsafeType': 'g-form',
         'failsafeUrl': ''
       },
-      fullscreenJourney: [],
-      questionJourney: [],
-      playbackJourney: [],
+      journey: [],
       hasVideoPlayed: -1,
-      sessionId: 1
+      sessionId: 1,
+      hasPlyrLoaded: false
```

Comment:
checks if `plyr` has loaded or not - it is needed as if someone presses fullscreen before plyr has loaded, currentTime is NaN